### PR TITLE
Create DanogoBond.json

### PR DESCRIPTION
**Pull request by thuannguyen2010**

### CODE REVIEW

File: dApps/DanogoBond.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            "contractLink": "https://github.com/Danogo2023/bonds-dex",
+            "contractId": "KDmn"
+        }
+    ],
+    "scripts": [
+        {
+            "id": "uXIBWd",
+            "name": "Swap Validator",
+            "purpose": "SPEND",
+            "type": "AIKEN",
```
Comment (matiwinnetou) [inline dApps/DanogoBond.json:79]: This is not properly handled in the backend now please write:

"type": "PLUTUS",
"contractLang": "AIKEN"

lang will be new optional field and I will fix other json files as well

PLUTUS does not refer in this case to the language but to the execution VM environment (PLUTUS Core)

CC: @clawoo

File: dApps/DanogoBond.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                    "scriptHash": "003a6fb5d426c5f1db82a40f3dad0435b745365f80881efe7bc89ec6",
+                    "contractAddress": "addr1wyqr5ma46snvtuwms2jq70ddqs6mw3fkt7qgs8h700yfa3sy07cvf"
+                }
+            ]
+        },
+        {
+            "id": "DyMyJK",
+            "name": "Swap Validator",
+            "purpose": "SPEND",
+            "type": "AIKEN",
```
Comment (matiwinnetou) [inline dApps/DanogoBond.json:129]: same as above

File: dApps/DanogoBond.json (hunk 1 above)
Comment (matiwinnetou) [reply dApps/DanogoBond.json:79]: Here is an example how it should look like: https://github.com/Cardano-Fans/crfa-offchain-data-registry/commit/f4f83b01c273ea63fe7a031a2edceb4c931a1dd1

As you can see there is a type field and a new contractLang field

**Comment by matiwinnetou** [comment]
Thank You for your contribution, I only left one comment, looks good.

Have you considered following in the future CIP-72 process as well?

CIP-72 will deprecate this repository in the long term.

File: dApps/DanogoBond.json (hunk 1 above)
Comment (thuannguyen2010) [reply dApps/DanogoBond.json]: I have updated. I saw a file as the reference but it was out of date :) Thank you.

File: dApps/DanogoBond.json (hunk 2 above)
Comment (thuannguyen2010) [reply dApps/DanogoBond.json]: I have updated

**Comment by thuannguyen2010** [comment]
> Thank You for your contribution, I only left one comment, looks good.
> 
> Have you considered following in the future CIP-72 process as well?
> 
> Combination of CIP-72 and certification CIP will deprecate this repository in the long term.

**Comment by thuannguyen2010** [comment]
> Thank You for your contribution, I only left one comment, looks good.
> 
> Have you considered following in the future CIP-72 process as well?
> 
> Combination of CIP-72 and certification CIP will deprecate this repository in the long term.

- I have read about CIP-72 but to be honest, I'm not clear about works that I have to do to follow this CIP.
- I need to create an off-chain metadata object, returning it by an REST API (or bit bucket). And I need to create an "anchored" on-chain data to refer to this off-chain data?
- And your repository has completed 2 steps for dApp? Now you say that CIP-72 and your certificate CIP are 2 seperate parts? And do I have to control and deploy to follow CIP-72 myself?
- I'm sorry if my question is naive or confused.

**Comment by matiwinnetou** [comment]
> > Thank You for your contribution, I only left one comment, looks good.
> > Have you considered following in the future CIP-72 process as well?
> > Combination of CIP-72 and certification CIP will deprecate this repository in the long term.
> 
> * I have read about CIP-72 but to be honest, I'm not clear about works that I have to do to follow this CIP.
> * I need to create an off-chain metadata object, returning it by an REST API (or bit bucket). And I need to create an "anchored" on-chain data to refer to this off-chain data?
> * And your repository has completed 2 steps for dApp? Now you say that CIP-72 and your certificate CIP are 2 seperate parts? And do I have to control and deploy to follow CIP-72 myself?
> * I'm sorry if my question is naive or confused.

1. Your questions are perfectly valid.
2. For the moment we support this registry, which is a centralised registry hosted on github
3. In the longer term we would like to move to CIP-72 registry only, there is a project in initial phases that will index data dApp data from Cardano on-chain and then provide this for DappsOnCardano.com (https://github.com/Cardano-Fans/crfa-dapp-registration-and-certification-service/)
4. Step by step, the way you have done this for now is correct and fine, CIP-72 is longer term. Initially we would ask people to try to provide both, legacy way and new way with CIP-72
5. Yes, you are correct, there is a small on-chain part in CIP-72 and also off-chain part. I do confess that this is quite bare bones at the moment, there is no tool that checks correctness or something. There are JSON Schemas and CDDLs though.

**Pull request merged**